app/i18n/es.json

```diff
@@ -505,6 +505,7 @@
     "seeTransaction": "Supervisar la transacción de apertura",
     "send": "Enviar",
     "title": "Mover dinero",
+    "updateAvailable": "Hay disponible una actualización.\nToca para actualizar ahora",
```

Comment:
we have a problem with our CTA's 😅

```suggestion
    "updateAvailable": "Hay una actualización disponible.\nToque para actualizar",
```

Reply:
Hi @leesalminen .  Can you accept @dolcalmi suggested change for the Spanish translation?